crates/turborepo-lib/src/run/summary/execution.rs

```diff
@@ -509,4 +528,19 @@ mod test {
             "tracker duration should be at least as long as the time between calls"
         );
     }
+
+    #[test_case(
+        TaskExecutionSummary {
+            started_at: 123,
+            ended_at: 234,
+            duration: TurboDuration::from(Duration::zero()),
+            state: ExecutionState::Built { exit_code: 0 },
+            exit_code: Some(0),
+        },
+        json!({ "startTime": 123, "endTime": 234, "exitCode": 0 })
```

Comment:
This is the expected `execution` schema as shown in [run-summary/monorepo.t](https://github.com/vercel/turbo/blob/main/turborepo-tests/integration/tests/run-summary/monorepo.t#L116)